### PR TITLE
Pagination: add Getting Started tutorial

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: concepts/05 UI Components/Pagination/05 Getting Started with Pagination/10 Configure Pagination.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+This section of the tutorial details the steps for basic Pagination setup.
```
Comment (Abadzhev) [inline concepts/05 UI Components/Pagination/05 Getting Started with Pagination/10 Configure Pagination.md]:
```suggestion
This tutorial step guides you through basic Pagination setup.
```

File: concepts/05 UI Components/Pagination/05 Getting Started with Pagination/10 Configure Pagination.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+First, assign a total number of items to the [itemCount](/Documentation/ApiReference/UI_Components/dxPagination/Configuration/#itemCount) property. Pagination will not function properly without this setting.
+
+Next, specify the [pageIndex](/Documentation/ApiReference/UI_Components/dxPagination/Configuration/#pageIndex) property to define the initial page to display. This tutorial sets **pageIndex** to 3 (the default value is 1).
+
+The [allowedPageSizes](/Documentation/ApiReference/UI_Components/dxPagination/Configuration/#allowedPageSizes) option is preset to `[5, 10]`. These are available page sizes. You can modify this list as needed. Include `'all'` to show every item on one page. In this tutorial, the default values are used.
+
+Finally, select an initial number of items to display per page from your allowed sizes. Assign the number to the [pageSize](/Documentation/ApiReference/UI_Components/dxPagination/Configuration/#pageSize) property.
```
Comment (Abadzhev) [inline concepts/05 UI Components/Pagination/05 Getting Started with Pagination/10 Configure Pagination.md]: I would turn this into a list. That way, we don't need to include words like "First", "Next", and "Finally", like there's some required sequence or there's some story to follow. It's just a few settings that you may want to specify. 

```suggestion
Specify the following settings:

* [itemCount](/Documentation/ApiReference/UI_Components/dxPagination/Configuration/#itemCount) sets the total number of items. Pagination will not function properly without this setting.
* [pageIndex](/Documentation/ApiReference/UI_Components/dxPagination/Configuration/#pageIndex) sets the initial page to display. This tutorial sets **pageIndex** to 3 (the default value is 1).
* [allowedPageSizes](/Documentation/ApiReference/UI_Components/dxPagination/Configuration/#allowedPageSizes) specifies page sizes available to users. Modify this list as needed. Include `'all'` to allow users to display all items on one page. This tutorial uses the default value: `[5, 10]`.
* [pageSize](/Documentation/ApiReference/UI_Components/dxPagination/Configuration/#pageSize) specifies the initial page size.
```

File: concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/00 Implement Remote Pagination.md [hunk 3]
```diff
@@ -0,0 +1,243 @@
+This section explains how to implement remote pagination.
```
Comment (Abadzhev) [inline concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/00 Implement Remote Pagination.md]:
```suggestion
This section explains how to implement remote pagination. Client code generates a color list and requests a remote service for cards that represent those color entries on screen. The pagination component helps users browse resulting color cards.
```

File: concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/00 Implement Remote Pagination.md [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+
+In this tutorial, colored cards load as a user changes pages or adjusts page size. The essential steps are the following:
```
Comment (Abadzhev) [inline concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/00 Implement Remote Pagination.md]:
```suggestion
Implementation can be broken down into three steps:
```

File: concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/00 Implement Remote Pagination.md [hunk 5]
```diff
@@ -0,0 +1,243 @@
...
+
+1. Generate 100 hex codes.
+2. Fetch [The Color API](https://www.thecolorapi.com/) (open source) whenever:
```
Comment (Abadzhev) [inline concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/00 Implement Remote Pagination.md]:
```suggestion
2. Fetch color cards from [The Color API](https://www.thecolorapi.com/) service when necessary:
```

File: concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/00 Implement Remote Pagination.md [hunk 6]
```diff
@@ -0,0 +1,243 @@
...
+    - The page initially loads
```
Comment (Abadzhev) [inline concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/00 Implement Remote Pagination.md]:
```suggestion
    - On page load
```

File: concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/00 Implement Remote Pagination.md [hunk 7]
```diff
@@ -0,0 +1,243 @@
...
+    - Page size changes
```
Comment (Abadzhev) [inline concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/00 Implement Remote Pagination.md]:
```suggestion
    - On page size changes
```

File: concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/00 Implement Remote Pagination.md [hunk 8]
```diff
@@ -0,0 +1,243 @@
...
+    - Page is switched (page index changes)
```
Comment (Abadzhev) [inline concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/00 Implement Remote Pagination.md]:
```suggestion
    - On page index changes
```

File: concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/00 Implement Remote Pagination.md [hunk 9]
```diff
@@ -0,0 +1,243 @@
...
+3. Display the fetched colored cards after loading.
```
Comment (Abadzhev) [inline concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/00 Implement Remote Pagination.md]:
```suggestion
3. Display color cards obtained from the service. 
```

File: concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/00 Implement Remote Pagination.md [hunk 10]
```diff
@@ -0,0 +1,243 @@
...
+
+Begin by declaring functions to generate random pastel hex codes and add them to an array of length 100:
```
Comment (Abadzhev) [inline concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/00 Implement Remote Pagination.md]:
```suggestion
Implement the first step. Generate 100 random pastel hex codes and add them to an array:
```

File: concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/10 Render Items.md [hunk 11]
```diff
@@ -0,0 +1,260 @@
+The render function determines the start and end indexes of the visible cards array. Next, the array populates with image sources and alt strings.
```
Comment (Abadzhev) [inline concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/10 Render Items.md]:
```suggestion
The render function determines the subset of cards to be displayed and populates the array with required images and alt strings.
```

File: concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/20 Integrate LoadPanel.md [hunk 12]
```diff
@@ -0,0 +1,215 @@
+Integrate a [load panel](/Documentation/Guide/UI_Components/LoadPanel/Overview/) to inform users that cards are loading. This step is optional.
```
Comment (Abadzhev) [inline concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/20 Integrate LoadPanel.md]: Other steps are not in imperative mood. They say "the code does this/that". 

```suggestion
The following code integrate a [load panel](/Documentation/Guide/UI_Components/LoadPanel/Overview/) into the application. The panel appears when the app requests card data from the remote service. This step is optional.
```

File: concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/20 Integrate LoadPanel.md [hunk 13]
```diff
@@ -0,0 +1,215 @@
...
+
+---
+##### jQuery
+
+To integrate DevExtreme LoadPanel:
```
Comment (Abadzhev) [inline concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/20 Integrate LoadPanel.md]:
```suggestion
To integrate the DevExtreme LoadPanel component:
```

File: concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/20 Integrate LoadPanel.md [hunk 14]
```diff
@@ -0,0 +1,215 @@
...
+
+1. Add LoadPanel to the code.
```
Comment (Abadzhev) [inline concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/20 Integrate LoadPanel.md]:
```suggestion
1. Add a LoadPanel to the code.
```

File: concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/20 Integrate LoadPanel.md [hunk 15]
```diff
@@ -0,0 +1,215 @@
...
+2. [Display](/Documentation/ApiReference/UI_Components/dxLoadPanel/Methods/#show) it before calling the render function.
+3. [Hide](/Documentation/ApiReference/UI_Components/dxLoadPanel/Methods/#hide) it after rendering each color.
```
Comment (Abadzhev) [inline concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/20 Integrate LoadPanel.md]:
```suggestion
3. [Hide](/Documentation/ApiReference/UI_Components/dxLoadPanel/Methods/#hide) it after render.
```

File: concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/20 Integrate LoadPanel.md [hunk 16]
```diff
@@ -0,0 +1,215 @@
...
+            })
+        .dxPagination('instance');
+        // ...
+        loadPanel.show();
+        renderCards(pageSize, pageIndex).finally(() => loadPanel.hide());
+    });
+
+##### Angular
+
+To integrate DevExtreme LoadPanel:
```
Comment (Abadzhev) [inline concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/20 Integrate LoadPanel.md]:
```suggestion
To integrate the DevExtreme LoadPanel component:
```

File: concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/20 Integrate LoadPanel.md [hunk 17]
```diff
@@ -0,0 +1,215 @@
...
+
+1. Add LoadPanel to the code.
```
Comment (Abadzhev) [inline concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/20 Integrate LoadPanel.md]:
```suggestion
1. Add a LoadPanel to the code.
```

File: concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/20 Integrate LoadPanel.md [hunk 18]
```diff
@@ -0,0 +1,215 @@
...
+2. [Display](/Documentation/ApiReference/UI_Components/dxLoadPanel/Configuration/#visible) it before calling the render function.
+3. Hide it after rendering each color.
```
Comment (Abadzhev) [inline concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/20 Integrate LoadPanel.md]:
```suggestion
3. Hide it after render.
```

File: concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/20 Integrate LoadPanel.md [hunk 19]
```diff
@@ -0,0 +1,215 @@
...
+                console.error('Error fetching colors:', error);
+            } finally {
+                this.loadPanelVisible = false;
+            }
+        }
+    }
+
+##### Vue
+
+To integrate DevExtreme LoadPanel:
```
Comment (Abadzhev) [inline concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/20 Integrate LoadPanel.md]:
```suggestion
To integrate the DevExtreme LoadPanel component:
```

File: concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/20 Integrate LoadPanel.md [hunk 20]
```diff
@@ -0,0 +1,215 @@
...
+
+1. Add LoadPanel to the code.
```
Comment (Abadzhev) [inline concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/20 Integrate LoadPanel.md]:
```suggestion
1. Add a LoadPanel to the code.
```

File: concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/20 Integrate LoadPanel.md [hunk 21]
```diff
@@ -0,0 +1,215 @@
...
+2. [Display](/Documentation/ApiReference/UI_Components/dxLoadPanel/Configuration/#visible) it before calling the render function.
+3. Hide it after rendering each color.
```
Comment (Abadzhev) [inline concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/20 Integrate LoadPanel.md]:
```suggestion
3. Hide it after render.
```

File: concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/20 Integrate LoadPanel.md [hunk 22]
```diff
@@ -0,0 +1,215 @@
...
+            loadPanelVisible.value = false;
+        }
+    };
+
+    // ...
+    </script>
+
+##### React
+
+To integrate DevExtreme LoadPanel:
```
Comment (Abadzhev) [inline concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/20 Integrate LoadPanel.md]:
```suggestion
To integrate the DevExtreme LoadPanel component:
```

File: concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/20 Integrate LoadPanel.md [hunk 23]
```diff
@@ -0,0 +1,215 @@
...
+
+1. Add LoadPanel to the code.
```
Comment (Abadzhev) [inline concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/20 Integrate LoadPanel.md]:
```suggestion
1. Add a LoadPanel to the code.
```

File: concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/20 Integrate LoadPanel.md [hunk 24]
```diff
@@ -0,0 +1,215 @@
...
+2. [Display](/Documentation/ApiReference/UI_Components/dxLoadPanel/Configuration/#visible) it before calling the render function.
+3. Hide it after rendering each color.
```
Comment (Abadzhev) [inline concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/20 Integrate LoadPanel.md]:
```suggestion
3. Hide it after render.
```

File: concepts/05 UI Components/Pagination/05 Getting Started with Pagination/00 Getting Started with Pagination.md [hunk 25]
```diff
@@ -0,0 +1,13 @@
+#include tutorials-intro-installationnote
+
+Pagination is a UI component that allows users to navigate through pages and change page size at runtime.
+
+This tutorial explains how to add Pagination to the page and configure the component's core settings. It also covers implementing remote pagination. Colored cards are loaded each time a user switches pages or changes page size. The final result is displayed below:
```
Comment (dirk-pieterse) [inline concepts/05 UI Components/Pagination/05 Getting Started with Pagination/00 Getting Started with Pagination.md]:
```suggestion
This tutorial explains how to add Pagination to a page and configure the component's core settings. It also covers implementing remote pagination. Colored cards are loaded each time a user switches pages or changes page size. The final result is displayed below:
```

File: concepts/05 UI Components/Pagination/05 Getting Started with Pagination/10 Configure Pagination.md [hunk 26]
```diff
@@ -0,0 +1,100 @@
+This tutorial step guides you through basic Pagination setup.
```
Comment (dirk-pieterse) [inline concepts/05 UI Components/Pagination/05 Getting Started with Pagination/10 Configure Pagination.md]:
```suggestion
This tutorial step guides you through the basic Pagination setup.
```

File: concepts/05 UI Components/Pagination/05 Getting Started with Pagination/10 Configure Pagination.md [hunk 27]
```diff
@@ -0,0 +1,100 @@
...
+
+Specify the following settings:
+
+* [itemCount](/Documentation/ApiReference/UI_Components/dxPagination/Configuration/#itemCount) sets the total number of items. Pagination will not function properly without this setting.
```
Comment (dirk-pieterse) [inline concepts/05 UI Components/Pagination/05 Getting Started with Pagination/10 Configure Pagination.md]:
```suggestion
* [itemCount](/Documentation/ApiReference/UI_Components/dxPagination/Configuration/#itemCount) sets the total number of items. Pagination does not function properly without this setting.
```

File: concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/00 Implement Remote Pagination.md [hunk 28]
```diff
@@ -0,0 +1,243 @@
+This section explains how to implement remote pagination. Client code generates a color list and requests a remote service for cards that represent those color entries on screen. The pagination component helps users browse resulting color cards.
```
Comment (dirk-pieterse) [inline concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/00 Implement Remote Pagination.md]:
```suggestion
This section explains how to implement remote pagination. Client code generates a color list and requests a remote service for cards representing color entries on the screen. The pagination component helps users browse the resulting color cards.
```

File: concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/10 Render Items.md [hunk 29]
```diff
@@ -0,0 +1,260 @@
+The render function determines the subset of cards to be displayed and populates the array with required images and alt strings.
```
Comment (dirk-pieterse) [inline concepts/05 UI Components/Pagination/05 Getting Started with Pagination/15 Implement Remote Pagination/10 Render Items.md]:
```suggestion
The render function determines the subset of cards to be displayed and populates the array with images and alt strings.
```

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**